dspace/config/spring/api/item-filters.xml

```diff
@@ -335,5 +346,27 @@
         </property>
     </bean>
 
+    <bean id="example-doi_filter" class="org.dspace.content.logic.DefaultFilter">
```

Comment:
Could we add a basic comment to this to describe what it might be used for?  Right now, it's here as an "example", but it's unclear (to me at least) why DOIs might want to use such a filter.  It'd be good to describe this a bit more in a code comment or similar.

(Or if this example isn't needed anymore, then we should remove it.  Just saying it doesn't make much sense right now without some sort of explanation)

Reply:
Good catch, I actually removed it in the end.